### PR TITLE
docs(components): Update Select docs

### PR DESCRIPTION
**Pull request by taylorvnoj**
## Motivations

Select mobile and web documentation needs to be combined. Shared states are on the docs page

### Added

- extra mobile story created to show initial value implementation
- I decided to include the Options props in the docs page; let me know your thoughts!

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: docs/components/Select/Web.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+export const InitialValue = InitialValueTemplate.bind({});
+InitialValue.args = {
+  placeholder: "Pick a friend",
+  defaultValue: "yoink",
+};
+
+export const Events = InitialValueTemplate.bind({});
+Events.args = {
+  onChange: (newValue: string) => {
+    console.log(newValue);
```
Comment (JeffMarvin) [inline docs/components/Select/Web.stories.tsx:92]: In exploring this example, the `console.log` is easy to miss... What do you think about `alert()`ing the `newValue` instead?

File: docs/components/Select/Web.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    <Option value="tony">Tony</Option>
+    <Option value="quincy">Quincy</Option>
+    <Option value="peppa">Peppa Pig</Option>
+  </Select>
+);
+
+const SizesTemplate: ComponentStory<typeof Select> = args => {
+  return (
+    <Content>
+      <Select size="small">
```
Comment (JeffMarvin) [inline docs/components/Select/Web.stories.tsx:34]: Clever & well done way to handle the args here for this "sticker sheet" type presentation! :)

**Review by JeffMarvin — CHANGES_REQUESTED** [review]
Needs;
more; and
semicolons!

Just had one suggestion/question. Nice stuff!

**Comment by taylorvnoj** [comment]
Thanks Jeff! Having an alert is much better than a `console.log` - pushed that change 😄

**Pull request merged**